x-pack/plugins/security/server/authentication/api_keys.test.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { createAPIKey } from './api_keys';
+import { loggingServiceMock } from '../../../../../src/core/server/mocks';
+
+const mockCallAsCurrentUser = jest.fn();
+
+beforeAll(() => jest.resetAllMocks());
+
+describe('createAPIKey()', () => {
+  test('returns null when security feature is disabled', async () => {
```

Comment:
__nit:__ sorry haven't noticed that during initial review: IIRC we use `it` instead of `test` in security